relay-server/src/services/processor.rs

```diff
@@ -1757,7 +1760,6 @@ impl EnvelopeProcessorService {
         //
         // Unconditionally scrub to make sure PII is removed as early as possible.
         event::scrub(&mut event, project_info.clone())?;
-        attachment::scrub(managed_envelope, project_info.clone());
```

Comment:
I think we need to keep this in until the option is removed and attachments are always removed.

Reply:
Yes, good catch!